src/game_battlealgorithm.cpp

```diff
@@ -773,8 +756,20 @@ int Game_BattleAlgorithm::AlgorithmBase::GetSourceAnimationState() const {
 
 void Game_BattleAlgorithm::AlgorithmBase::TargetFirst() {
 	if (party_target) {
-		party_target->GetBattlers(targets);
+		if (reflect_set) {
+			PlayAnimation();
```

Comment:
We should not be playing animations here. This function only changes the targets and returns the first one.